.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,8 @@
+### Checklist
```

Comment:
Are you still cool with the `### Description` lower down? I think that one's more important, to break things up.